test/axes/numericAxisTests.ts

```diff
@@ -16,453 +16,438 @@ describe("NumericAxis", () => {
+    it("separates tick labels with the same spacing", () => {
+      let scale = new Plottable.Scales.Linear();
+      scale.domain([-2500000, 2500000]);
+
+      let axis = new Plottable.Axes.Numeric(scale, "bottom");
+      axis.renderTo(svg);
+
+      let visibleTickLabels = axis.content().selectAll(".tick-label")
         .filter(function(d: any, i: number) {
           let visibility = d3.select(this).style("visibility");
           return (visibility === "visible") || (visibility === "inherit");
         });
-    assert.isTrue(tickLabels[0].length > 1, "more than one tick label is shown");
 
-    for (let i = 0; i < tickLabels[0].length - 1; i++) {
-      let currLabel = d3.select(tickLabels[0][i]);
-      let nextLabel = d3.select(tickLabels[0][i + 1]);
-      assert.isTrue(Number(currLabel.text()) > Number(nextLabel.text()), "numbers are arranged in descending order from left to right");
-    }
+      let visibleTickLabelRects = visibleTickLabels[0].map((label: Element) => label.getBoundingClientRect());
+
+      function getClientRectCenter(rect: ClientRect) {
+        return rect.left + rect.width / 2;
+      }
 
-    svg.remove();
+      let interval = getClientRectCenter(visibleTickLabelRects[1]) - getClientRectCenter(visibleTickLabelRects[0]);
+      d3.pairs(visibleTickLabelRects).forEach((rects, i) => {
+        assert.closeTo(getClientRectCenter(rects[1]) - getClientRectCenter(rects[0]),
+          interval, 0.5, `tick label pair ${i} is spaced the same as the first pair`);
+      });
+
+      svg.remove();
+    });
+
+    it("renders numbers in order with a reversed domain", () => {
+      let scale = new Plottable.Scales.Linear();
+      scale.domain([3, 0]);
+
+      let axis = new Plottable.Axes.Numeric(scale, "bottom");
+      axis.renderTo(svg);
+
+      let tickLabels = axis.content().selectAll(".tick-label")
+          .filter(function(d: any, i: number) {
+            let visibility = d3.select(this).style("visibility");
+            return (visibility === "visible") || (visibility === "inherit");
+          });
+      assert.operator(tickLabels.size(), ">", 1, "more than one tick label is shown");
+
+      let tickLabelElementPairs = d3.pairs(tickLabels[0]);
+      tickLabelElementPairs.forEach(function(tickLabelElementPair, i) {
+        let label1 = d3.select(tickLabelElementPair[0]);
+        let label2 = d3.select(tickLabelElementPair[1]);
+        let labelNumber1 = parseFloat(label1.text());
+        let labelNumber2 = parseFloat(label2.text());
+        assert.operator(labelNumber1, ">", labelNumber2, `pair ${i} arranged in descending order from left to right`);
+      });
+
+      svg.remove();
+    });
   });
 
-  it("constrained tick labels do not overlap tick marks", () => {
-    let svg = TestMethods.generateSVG(100, 50);
+  describe("drawing tick labels when left oriented", () => {
+    it("draws ticks labels centered with the corresponding tick mark", () => {
+      let svg = TestMethods.generateSVG();
```

Comment:
there's a couple duplicated lines, can you move them to beforeEach?
(can be consistent with `numericAxis` or `axis`)